turborepo-tests/integration/tests/bad-turbo-json.t

```diff
@@ -13,15 +13,15 @@ Run build with package task in non-root turbo.json
         unnecessary-package-task-syntax)
         
           x "my-app#build". Use "build" instead.
-            ,-\(apps(\/|\\)my-app(\/|\\)turbo.json:8:21\) (re)
-          7 |         // this comment verifies that turbo can read .json files
+            ,-\(apps(\/|\\)my-app(\/|\\)turbo.json:6:21\) (re)
+          5 |         // this comment verifies that turbo can read .json files
```

Comment:
Only needed to be updated because this snapshot includes line numbers and we removed bad lines from the fixture